README.rst

```diff
@@ -23,6 +23,30 @@ This library provides a library called `thoth-storages
 for `PostgreSQL database <https://www.postgresql.org/>`__ as well as adapters
 for manipulating with `Ceph <https://ceph.io/>`__ via its S3 compatible API.
 
+Quick Start
+===========
+
+Pre-requisites:
+
+* make sure you have ``podman-compose`` installed
+* make sure you are in an environment created with ``pipenv install --dev``
+
+To develop locally the first time:
+
+* Have a pg dump that you can `retrieve from aws s3
+<https://github.com/thoth-station/storages#automatic-backups-of-thoth-deployment>`__
+* Run ``podman-compose up`` to scale up pods for database and pgweb
```

Comment:
Here I would also refer to the section Running PostgreSQL locally for more details if they have to

Reply:
Thanks, changes have been added :+1: